osquery/main/windows/daemon.cpp

```diff
@@ -82,9 +211,26 @@ void WINAPI ServiceControlHandler(DWORD control_code) {
   }
 }
 
-void daemonEntry(int argc, char *argv[]) {
+void daemonEntry(int argc, char* argv[]) {
   osquery::Initializer runner(argc, argv, osquery::ToolType::DAEMON);
 
+  // Options for installing or uninstalling the osqueryd as a service
+  if (osquery::FLAGS_install) {
+    Status s = osquery::installOsqueryService(std::string(argv[0]));
+    if (!s.ok()) {
```

Comment:
If you're not doing anything with the `Status` code/message then `if(osquery::installOsqueryService) {` saves code.